### DISCUSSION TITLE
elasticsearch log grows to quick

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#3500] [Unsupported Versions]
I am running into an issue where the elasticsearch logs can grow to take up the whole 2 TB hd over several hours. This is probably because of the amount of alerts that are coming through i would assume. I configured it with suricata and  snort oinkcode.   I have had issues with disabling rules even with the new so-rule-update -disable command. Even though it says they are disabled they still come through, possibly because they are gid 3 not gid 1? Is there another way to cut back on logs so that i wont fill up the hd in hours causing it stop logging new alerts ? I have yet to be successfully at disabling/suppressing overly active alerts. If i am able to get that fixed i believe it will help with reducing elasticsearch from filling up so quick.

**Comment by TOoSmOotH** [comment]
This would be from metadata, not alerts that are filling up your ES instance. You can see what is taking up the most space by running the following on wherever your data is being stored:

curl localhost:9200/_cat/indices if it's a standalone.
curl -sk https://localhost/_cat/indices if it's a search node. 

You would need to use zeek or suricata filters to control the amount of data coming in.

**Reply by wtl06639** [reply]
Is there a way to reduce the amount of metadata for each alert?
